### DISCUSSION TITLE
Category Closure: Machine Learning 🧠

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#69839] [Programming Help]
In April, [we announced the trial of two new categories in the community](https://github.com/orgs/community/discussions/52655), one of them being the Machine Learning and ML Ops category. These categories were launched on a trial basis which concluded on September 30. Based on the data we reviewed (user engagement, activity, views, etc.) and our experience we will be closing the Machine Learning and MLOps category. 

At GitHub we employ a ship to learn mindset and we believe that applies in our community too. We appreciate having a community full of folks who allow us to experiment based on what trends were seeing from them and the world around us 🌍 . 

So what's next for the Machine Learning category? The category will be closed effective October 11, however, we're excited to share the Programming Help category is here to stay 🎉 . All posts from the Machine Learning category will be moved to our [Programming Help category](https://github.com/orgs/community/discussions/categories/programming-help) and the labels that launched with the categories will remain so please still use them on your posts if they relate to `Machine Learning`, `AI or ML Model` or `MLOps` posts. We still want to chat about Machine Learning, but we've learned it may no need its own category and conversations can continue in Programming Help

Thanks to all who participated in the trials of both categories. We're always keen to improve so feel free to share any feedback in the comments below. For any requests for a new category or idea for the community, [let us know in our latest Community Survey](https://github.com/orgs/community/discussions/69811).

**Comment by Bouhrir** [comment]
The decision to close the Machine Learning and MLOps category on GitHub is understandable based on the data and experience reviewed. It’s important to allocate resources efficiently and focus on categories that generate significant engagement and activity.

**Comment by bizactive** [comment]
For anyone interested in understanding ML check out https://www.udacity.com/
